app/Models/News.php

```diff
@@ -48,11 +48,11 @@ class News extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->